### PR TITLE
fix: forward context in common's status pkg

### PR DESCRIPTION
**Pull request by filariow**
related to [#352](https://github.com/codeready-toolchain/toolchain-common/pull/352)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -114,3 +114,5 @@ require (
 )
 
 go 1.20
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/filariow/toolchain-common v0.0.0-20240109160929-447985913c7f
```
Comment (xcoulon) [inline go.mod:118]: reminder

**Review by xcoulon — APPROVED** [review]

**Review by alexeykazakov — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (filariow) [reply go.mod:118]: thx

Comment (alexeykazakov) [reply go.mod:118]: We don't really have to use any reminders anymore. We have our `go.mod replacment` check which automatically blocks all PRs with `raplace` in it :)
See https://github.com/codeready-toolchain/host-operator/actions/runs/7463977391/job/20309889472?pr=955

**Pull request merged**